src/Sfa.Tl.ResultsAndCertification.Application/Interfaces/ICommonService.cs

```diff
@@ -27,5 +28,7 @@ public interface ICommonService
         bool IsIndustryPlacementTriggerDateValid();
        
         DateTime CurrentDate { get; }
+
+        Task<bool> AddChangelog(Domain.Models.ChangeLog changeLog);
```

Comment:
There is an alias in the using's called Domain.

Reply:
Updated as per comments